src/Pages/settings/Relays.tsx

```diff
@@ -20,6 +20,8 @@ const RelaySettingsPage = () => {
     const ev = await publisher.saveRelays();
     publisher.broadcast(ev);
     publisher.broadcastForBootstrap(ev);
+    const settingsEv = await publisher.saveRelaysSettings();
```

Comment:
Maybe we can write to all (or a lot of relays) from `https://api.nostr.watch/v1/online`

Reply:
Sending to all of user relays and random sample of 20 from that endpoint.